app/services/update_extracted_text.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+# @abstract Updates the extracted text for a give FileResource.
+class UpdateExtractedText
+  def self.call(resource:, force: false)
+    return if resource.extracted_text.present? && (force != true)
```

Comment:
I guess we explicitly have to say `force != true` to protect against passing some random non-boolean value? Otherwise we could just use `!force`.

Reply:
I like to be explicit here. There are some cases where `"false"` would evaluate as `true`. Ex:

``` ruby
> force = "false"
=> "false"
> puts "hi" if force
hi
```

But,

``` ruby
> puts "hi" if force == true
=> nil
```

So while `"false"` will _sometimes_ be, I err on the side of caution and allow the only possible true case, where you definitely use a TrueClass.



It doesn't read very well, but the user will _have_ to say `force: true` otherwise, putting `force: "false"` would  evaluate to true.